taichi/common/core.cpp

```diff
@@ -26,6 +26,22 @@ __asm__(".symver log2f,log2f@GLIBC_2.2.5");
 float __wrap_log2f(float x) {
```

Comment:
nit: Let's move these wrappers to a separate file to hightlight their importance

Reply:
how about to a `symbol_version.h`?

Reply:
nit: `symbol_version.cpp` (IIUC, we only need these function to be defined)?

Reply:
> Can we also add link to the issue in both files? ;)

you mean link to this issue? #3174